src/data/faq.json

```diff
@@ -317,7 +317,7 @@
                         "active": true,
                         "textblock": [
                             "The app is currently available in the following languages:<ul><li>German</li><li>English</li><li>Turkish (as of version 1.1.1)</li><li>Bulgarian (as of version 1.2.0)</li><li>Polish (as of version 1.2.0)</li><li>Romanian (as of version 1.2.0)</li></ul>",
-                            "The telephone hotline (<a href='tel:+498007540001'>+49 800 7540001</a>) answers in Turkish, English and German. More languages are to come. We'll keep you up to date.", "The language of the app is derived from the language that is configured in the smartphone's language settings. To display the app in German, for example, change the language settings to German.",
+                            "The telephone hotline (<a href='tel: 08007540001'> 0800 7540001</a>) answers in Turkish, English and German. More languages are to come. We'll keep you up to date.", "The language of the app is derived from the language that is configured in the smartphone's language settings. To display the app in German, for example, change the language settings to German.",

```

Comment:
You need to remove the trailing space before 0800 so it will be tel:0800xxx

Reply:
same for >0800

Reply:
ok